security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -434,7 +434,7 @@ func (p *Plugin) fetchAccessToken(federatedToken *federatedTokenResponse) (*acce
 	} else {
 		tokenExp = exp
 	}
-	pluginLog.WithLabels("latency", timeElapsed.String(), "ttl", tokenExp).Infof("fetched access token")
+	pluginLog.WithLabels("latency", timeElapsed.String(), "ttl", -time.Since(tokenExp)).Infof("fetched access token")
```

Comment:
For better readability, consider replacing "-time.Since(tokenExp)" with time.Until(tokenExp).